pkg/kubelet/cm/cgroup_manager_linux_test.go

```diff
@@ -126,6 +126,39 @@ func TestCgroupNameToSystemd(t *testing.T) {
 	}
 }
 
+func TestCgroupNameWithEmptyPartToSystemd(t *testing.T) {
+	testCases := []struct {
+		input    CgroupName
+		expected string
+	}{
+		{
+			input:    RootCgroupName,
+			expected: "/",
+		},
+		{
+			input:    NewCgroupName(RootCgroupName, "Burstable", ""),
+			expected: "/Burstable.slice",
+		},
+		{
+			input:    NewCgroupName(RootCgroupName, "", "Burstable", "pod-123"),
+			expected: "/Burstable.slice/Burstable-pod_123.slice",
+		},
+		{
+			input:    NewCgroupName(RootCgroupName, "BestEffort", "", "pod-6c1a4e95-6bb6-11e6-bc26-28d2444e470d", ""),
+			expected: "/BestEffort.slice/BestEffort-pod_6c1a4e95_6bb6_11e6_bc26_28d2444e470d.slice",
+		},
+		{
+			input:    NewCgroupName(RootCgroupName, "kubepods", ""),
+			expected: "/kubepods.slice",
+		},
+	}
+	for _, testCase := range testCases {
+		if actual := testCase.input.ToSystemd(); actual != testCase.expected {
```

Comment:
`require.Equal(t, testCase.expected, testcase.input.ToSystemd()` would be easier to read here and produce skimmable diffs

Reply:
@endocrimes Updated code as you suggested. Thanks!